front_end/src/components/indiv-hotel/IndivHotel.js

```diff
@@ -6,11 +6,36 @@ import { connect } from "react-redux";
 import SearchWidget from "../landing_page/search_widget/SearchWidget";
 import { Link } from "react-router-dom";
 import { saveBooking } from "../../actions/bookingActions";
-import { getIndividualHotelResult } from "../../actions/searchResultActions";
+import { getIndividualHotelResult, clearData } from "../../actions/searchResultActions";
 import moment from "moment";
+import Button from "@material-ui/core/Button";
+import {
+  Grid,
+  CircularProgress
+} from "@material-ui/core";import {
+  Typography,
+  ExpansionPanel,
+  ExpansionPanelDetails,
+  ExpansionPanelSummary,
+  Divider,
+} from "@material-ui/core";
+import {
+  Search,
+  ExpandMore,
+} from "@material-ui/icons";
+
+import "./indivHotel.css";
 import taxrates from "../payment/taxrates.json";
 
 class IndivHotel extends Component {
+  componentDidMount() {
+    window.scrollTo(0, 0);
+  }
+
+  componentWillUnmount(){
```

Comment:
if we clear the data, we won't be able to use the browsers back button anymore...